backend/foodbank_southlondon/templates/day-overview.html

```diff
@@ -118,6 +123,9 @@ <h1>DAY OVERVIEW</h1>
                     <td>
                         <span class="no-wrap">
                             {{ request['postcode'] }}
+                            {% if request['congestion_zone'] %}
```

Comment:
Actually, I think you can just do {{ url_for('static', filename='congestion-charge.png', _external=True)}} and then we just need to make sure that frontend/src/assets/congestion-charge.png appears in frontend/public/. Dan did this for logo.png.